modules/client-configuration/pages/clients-sleses.adoc

```diff
@@ -175,13 +175,13 @@ Mount the {rhel} or {centos} 6, 7 or 8 Media:
 Replace [command]``<os_name>`` with either ``rhel``  or ``centos``:
```

Comment:
... or ``sleses`` (Expanded Support), or ``ol`` (Oracle Linux), and version should also be specified. E. g. ``sleses8``.

Given that this page is for SLES ES, I wouldn't menion RHEL or CentOS but say something like:

```suggestion
Replace [command]``<os_name>`` with either ``sleses6``, ``sleses7``  or ``sleses8``.
```

We should have this same content in the RHEL and CentOS doc pages.